src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -8198,7 +8209,7 @@ internal void TdsLogin(SqlLogin rec, TdsEnums.FeatureExtension requestedFeatures
                     }
                     if ((requestedFeatures & TdsEnums.FeatureExtension.DataClassification) != 0)
                     {
-                        length += WriteDataClassificationFeatureRequest(true);
+                        WriteDataClassificationFeatureRequest(true);
```

Comment:
Though thi is not related to this PR, line 8195 has the same unneccessary length increasement ` length += WriteSessionRecoveryFeatureRequest(recoverySessionData, true);`

Reply:
Fixed. Also noticed a missing trace line.